packages/now-routing-utils/src/superstatic.ts

```diff
@@ -120,10 +127,14 @@ export function convertTrailingSlash(enable: boolean, status = 308): Route[] {
 }
 
 export function sourceToRegex(
-  source: string
+  source: string,
+  trailingSlash: boolean | undefined
 ): { src: string; segments: string[] } {
   const keys: Key[] = [];
-  const r = pathToRegexp(source, keys, { strict: true });
+  const r = pathToRegexp(source, keys, {
+    strict: !trailingSlash,
```

Comment:
```suggestion
    strict: trailingSlash === false ? true : false,
```

Consider the case, `trailingSlash` is undefined. We don't want to be strict because both should work

Reply:
Next.js doesn't support `trailingSlash` so I'm reverting this change until they do.